twilio-authenticator-sample/src/main/java/com/twilio/authenticatorsample/approvalrequests/RequestsFragment.java

```diff
@@ -54,18 +55,19 @@
     private MessageHelper messageHelper;
 
     private ApprovalRequests approvalRequests;
-
-    public RequestsFragment() {
-        // Required empty public constructor
-    }
+    private Long appId;
 
     /**
      * Use this factory method to create a new instance of Requests fragment
      *
      * @return A new instance of fragment RequestsFragment.
      */
-    public static RequestsFragment newInstance() {
-        return new RequestsFragment();
+    public static RequestsFragment newInstance(Long appId) {
```

Comment:
I'd rename this fragment to `ApprovalRequestsFragment` for consistency and domain naming.

Reply:
fixed